docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -48,6 +48,27 @@
             - "hostname: {{get_host_name.stdout}}"
             - "dc_role: {{ get_distributed_cloud_role.stdout }}"
 
+    - block:
+      # Configure required region value for subcloud
+        - name: Get subcloud region value from environment
+          shell: grep OS_REGION_NAME /etc/platform/openrc | awk -F"=" '{print $2}'
+          register: os_region_name_from_env
+
+        - name: Get subcloud region value from deployment file
+          shell: grep OS_REGION_NAME {{deploy_config}} | sed 's/^[ \t]*//'
+          register: os_region_name_from_deploy_file
+
+        - name: Configure new subcloud region value
```

Comment:
Please use ansible builtin regex module than the sed

Reply:
Done